doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,590 @@
+
+  * Fixed an issue where it was not possible to subscribe to the service notifications more than once.
+
+* Updated the :ref:`central_uart` sample to handle data packets that are longer than 212 bytes.
+  Enabled UART flow control to avoid data loss.
+
+* Enabled UART flow control in the :ref:`peripheral_uart` sample to avoid data loss.
+
+* Changed the :ref:`ble_throughput` sample to prevent it from running Bluetooth LE scanning and advertising in parallel.
+  The feature to establish a connection in both master and slave role at the same time is not supported by the Zephyr Bluetooth LE Host.
+
+* :ref:`nrf_bt_scan_readme`:
+
+  * Added an option to update the initial connection parameters.
+
+* :ref:`gatt_dm_readme`:
+
+  * Fixed an issue where service or characteristic allocation failed, but the returned pointer was not checked before accessing the data it pointed to.
+
+
+NFC
+---
+
+* Added the following samples:
+
+  * :ref:`nfc_tnep_tag` and :ref:`nfc_tnep_poller` -  demonstrate how to use the Tag NDEF Exchange Protocol (TNEP).
+  * :ref:`nrf-nfc-system-off-sample` - demonstrates how to wake the device from System OFF mode using NFC.
+    This sample was ported from the nRF5 SDK.
+
+* Added the following libraries:
+
+  * :ref:`nfc_t4t_cc_file_readme` - reads and parses the Capability Container file that can be found in the Type 4 Tag.
+  * :ref:`nfc_t4t_hl_procedure_readme` - performs the NDEF detection procedure for the Type 4 Tag.
+  * :ref:`tnep_tag_readme` - implements the Tag NDEF Exchange Protocol (TNEP) for a Tag device.
+  * :ref:`tnep_poller_readme` - implements the Tag NDEF Exchange Protocol (TNEP) for a Poller device.
+
+* Updated the NFC samples to enable stack protection, assertions, and logging by default.
+* Extended the :ref:`nfc_tag_reader` sample with parsing and printing of the Type 4 Tag content.
+* Moved the NFC Platform implementation to the fw-nrfconnect-nrf repository.
+  See :ref:`nrfxlib:nfc_integration_notes`.
+
+Multi-Protocol Service Layer (MPSL)
+-----------------------------------
+
+* Added MPSL as a new subsystem.
+  It integrates :ref:`nrfxlib:mpsl` into the |NCS| environment.
+
+* Added the following sample:
+
+  * :ref:`timeslot_sample` - demonstrates how to use :ref:`nrfxlib:mpsl` and basic MPSL Timeslot functionality.
+
+Setting storage
+---------------
+
+* Reduced the default partition size for the settings storage from 24 kB (0x6000) to 8 kB (0x2000).
+  This leaves more flash space to the application.
+
+nRF Desktop
+===========
+
+* Added a ``ble_qos`` module to maintain channel maps.
+
+Build system
+============
+
+* Fixed a bug where a user-defined HEX file that was provided in the static configuration of the :ref:`partition_manager` was not included in the merge operation.
+
+nrfx
+====
+
+* Updated to v2.1.0.
+  For details, see the `changelog <https://github.com/NordicSemiconductor/nrfx/blob/v2.1.0/CHANGELOG.md>`_.
+
```

Comment:
@ru-fu In order to keep this simple I will refer to the upstream revisions:

```
Zephyr
======

This nRF Connect SDK released is based on Zephyr 2.1.99 (and in particular on Zephyr SHA 40175fd3bd), which is revision sitting
in between the upstream Zephyr 2.1 and 2.2 releases. 

To see a comprehensive list of changes introduced since nRF Connect 1.1.0 you can use the following Git command:

.. code-block:: console
   git log 7d7fed0d2b..40175fd3bd
```